src/wcs2_0/src/test/java/org/geoserver/wcs2_0/kvp/GetCapabilitiesTest.java

```diff
@@ -168,4 +170,29 @@ public void testMetadata() throws Exception {
                 base + "ows:Metadata[2]/@xlink:href",
                 dom);
     }
+
+    @Test
+    public void testAcceptLanguagesParameter() throws Exception {
+        Catalog catalog = getCatalog();
```

Comment:
Are the catalog updates actually needed? I'd expect the machinery to work just fine without, one might have not setup a i18n in the title/description, but still use the language parameter in SLD files.

Reply:
Catalog and FeatureTypeInfo are needed to have language links to appear.

Reply:
Confirmed, otherwise an exception is thrown because nothing in the config has the requested language